iOS/DuckDuckGo/PullToRefreshViewAdapter.swift

```diff
@@ -80,6 +80,13 @@ final class PullToRefreshViewAdapter: NSObject {
     private weak var pullableView: UIView?
     private let onRefresh: () -> Void
 
+    var backgroundColor: UIColor? {
+        didSet {
+            fakeScrollView.backgroundColor = backgroundColor ?? UIColor(designSystemColor: .background)
+            refreshControl.tintColor = .red
```

Comment:
this is just for now, so the refreshControl can be seen on black background, we will need to calculate the color here